docs/usage/nextjs.mdx

```diff
@@ -63,7 +63,7 @@ These recommendations are specific to applications written with the Next.js App
 
 ### Folder Structure
 
-Next apps can be created to have the `/app` folder either at the root, or nested under `/src/app`. Your Redux logic should go in a separate folder, alongside the `/app` folder. It's common to put the Redux logic in a folder named `/lib`, but not required.
+Next apps can be created to have the `/app` folder either at the root, or nested under `/app`. Your Redux logic should go in a separate folder, alongside the `/app` folder. It's common to put the Redux logic in a folder named `/lib`, but not required.
```

Comment:
```suggestion
Next apps can be created to have the `/app` folder either at the root, or nested under `/src/app`. Your Redux logic should go in a separate folder, alongside the `/app` folder. It's common to put the Redux logic in a folder named `/lib`, but not required.
```
this isn't a typo, NextJS does support putting your app directory under a src directory: https://nextjs.org/docs/app/building-your-application/configuring/src-directory

The codeblock title makes sense to adjust as in most cases it won't be, but this sentence is correct as is.

Reply:
Yep, that part was intentional.